lightning/restore/restore.go

```diff
@@ -622,6 +624,8 @@ func (rc *RestoreController) restoreTables(ctx context.Context) error {
 			}
 			if cp.Status <= CheckpointStatusMaxInvalid {
 				allInvalidCheckpoints[tableName] = cp.Status
+			} else if cp.TableID > 0 && cp.TableID != tableInfo.ID {
```

Comment:
How to guarantee TableID is different in two time create tables?
it seems create table is in random order since we use map [here](https://github.com/pingcap/tidb-lightning/blob/c10636ed08e055fb1faba587fbe4293e8f4293c3/lightning/restore/tidb.go#L83)

Reply:
1. start Lightning with checkpoint enabled
2. use failpoint to stop Lightning
3. drop the table (and maybe create it again)
4. resume Lightning